extensions/wikia/Qualaroo/scripts/Qualaroo.js

```diff
@@ -4,9 +4,11 @@
 		createCookie;
 
 	createCookie = function(cookieName) {
-		var cookieValue = cookieName + '=true;path=/;domain=';
-		if (window.location.host.indexOf('wikia-dev')) {
-			cookieValue += '.wikia-dev.com';
+		var cookieValue = cookieName + '=true;path=/;domain=',
+			hostName = window.location.host,
+			devDomainIndex = hostName.indexOf('wikia-dev');
+		if (devDomainIndex) {
```

Comment:
In case the wikia-dev is not found, the devDomainIndex will equal -1. So we will never enter the else clause.